test/e2e/validation_test.go

```diff
@@ -309,9 +309,10 @@ func (tc *testContext) sshSetup() error {
 // PowerShell, on the host specified by the provided IP.
 func (tc *testContext) runPowerShellSSHJob(name, command, ip string) (string, error) {
 	// Modify command to work when default shell is the newer Powershell version present on Windows Server 2022.
-	// We only test Windows Server 2022 if a custom VXLAN port is used (i.e. vSphere)
+	// We only test Windows Server 2022 in vSphere and AWS
 	powershellDefaultCommand := command
-	if tc.hasCustomVXLAN {
+	if tc.CloudProvider.GetType() == config.VSpherePlatformType ||
+		tc.CloudProvider.GetType() == config.AWSPlatformType {
```

Comment:
I have a feeling this logic won't suffice if we are testing both 2019 and 2022 using AWS jobs. One suggestion is to replace `tc.hasCustomVXLAN` with a flag `isWindowsServer2022` in the [`testContext` struct](https://github.com/openshift/windows-machine-config-operator/blob/master/test/e2e/main_test.go#L64), setting it at the time of [machineset creation](https://github.com/openshift/windows-machine-config-operator/blob/2dfa5ee1d5b6223c11c6c27353b7429e99253b23/test/e2e/create_test.go#L262) in the e2e test suite. Might require some parsing of the machine set spec to see the OS version of the image being used.